src/commands/dev/dev.js

```diff
@@ -599,6 +616,12 @@ const createDevCommand = (program) => {
         .choices(['cache', 'mock', 'update'])
         .default('cache'),
     )
+    .addOption(
+      new Option(
+        '--country <geoCountry>',
+        'Two-letter country code ([ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements)) to use as mock geolocation (to be used with --geo=mock)',
```

Comment:
We do not have markdown here.

```suggestion
        'Two-letter country code (ISO 3166-1 alpha-2, https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) to use as mock geolocation (enables --geo=mock autmatically)',
```